<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>220</height>
   </size>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip {
  background-color: #148CD2;
  border: 1px solid #19232D;
  color: #C6C8D1;
  padding: 0px;
}</string>
  </property>
  <widget class="QFrame" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>220</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>220</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(28, 29, 32);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="btn_run">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
border-radius: 50;
color: rgb(208, 208, 208);
background-color: rgb(40, 44, 52);
 }

QToolButton::hover
{
    background-color: rgb(64, 70, 83);
}

QToolButton::checked
{
    background-color: rgb(85, 170, 127);
	color: rgb(149, 0, 0);
}
QToolButton::checked::hover
{
    background-color: rgb(0, 211, 102);
	color: rgb(170, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_dev">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>190</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="toolTip">
     <string>Follow Me!</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(78, 86, 102);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Devloped by: &lt;a href=&quot;https://www.instagram.com/9_tay&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#737f96;&quot;&gt;@9_tay&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Exit</string>
    </property>
    <property name="styleSheet">
     <string notr="true">



QToolButton {
border-radius: 8;
background-color: rgb(170, 0, 0);
 }

QToolButton::hover
{
    background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_min">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>System tray</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
border-radius: 8;
background-color: rgb(170, 85, 0);
 }

QToolButton::hover
{
    background-color: rgb(255, 170, 0);
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar
{	color: rgb(40, 44, 52);
}

QProgressBar:horizontal
{
    background-color: #626568;
    border: 0.1ex solid #31363b;
    border-radius: 0.3ex;
    height: 0.5ex;
    text-align: center;
    margin-top: 0.5ex;
    margin-bottom: 0.5ex;
    margin-right: 0ex;
    padding: 0px;
}

QProgressBar::chunk:horizontal
{
    background-color: #3daee9;
    border: 0.1ex transparent;
    border-radius: 0.3ex;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_count">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>195</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="toolTip">
     <string>Counter</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(90, 99, 117);
}</string>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>195</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="toolTip">
     <string>Status</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(90, 99, 117);
}
</string>
    </property>
    <property name="text">
     <string>-----</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>5</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(78, 86, 102);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://www.instagram.com/9_tay&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#4a5261;&quot;&gt;IG Downloader v2.1&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_info">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Info</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
border-radius: 8;
background-color: rgb(0, 85, 127);
 }

QToolButton::hover
{
    background-color: rgb(0, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>!</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_folder">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>10</y>
      <width>14</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Download Folder</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
border-radius: 8;
background-color: transparent;
 }

QToolButton::hover
{
    background-color: rgb(249, 190, 44);
}</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/folder.png</normaloff>:/folder.png</iconset>
    </property>
    <property name="shortcut">
     <string>Ctrl+D</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonFollowStyle</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_update">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Status</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: rgb(175, 5, 0);
    border-radius: 8;
	background-color: rgb(190, 200, 0);
}

QLabel::hover
{
    color: rgb(205, 0, 0);
	 background-color: rgb(200, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>Please don't exit until update completed</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_cookie">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>7</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Cookies</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
border-radius: 8;
background-color: transparent;
 }

QToolButton::hover
{
	
	
	
	background-color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/cookie.png</normaloff>:/cookie.png</iconset>
    </property>
    <property name="shortcut">
     <string>Ctrl+D</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonFollowStyle</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
